src/ripple/overlay/impl/PeerSet.cpp

```diff
@@ -104,47 +121,26 @@ void PeerSet::invokeOnTimer ()
         setTimer ();
 }
 
-bool PeerSet::isActive ()
-{
-    ScopedLockType sl (mLock);
-    return !isDone ();
-}
-
 void PeerSet::sendRequest (const protocol::TMGetLedger& tmGL, std::shared_ptr<Peer> const& peer)
 {
-    if (!peer)
-        sendRequest (tmGL);
-    else
-        peer->send (std::make_shared<Message> (tmGL, protocol::mtGET_LEDGER));
-}
+    auto packet = std::make_shared<Message>(tmGL, protocol::mtGET_LEDGER);
+
+    if (peer)
+    {
+        peer->send(packet);
+        return;
+    }
 
-void PeerSet::sendRequest (const protocol::TMGetLedger& tmGL)
-{
     ScopedLockType sl (mLock);
 
     if (mPeers.empty ())
         return;
 
-    auto packet = std::make_shared<Message>(tmGL, protocol::mtGET_LEDGER);
-
     for (auto id : mPeers)
     {
         if (auto peer = app_.overlay ().findPeerByShortID (id))
```

Comment:
Post-refactoring, `auto peer` will shadow the function argument. Consider using `p`.